examples/meta/src/evaluation/cross_validation_multiple_kernel_learning_weights_storage.sg

```diff
@@ -32,7 +32,7 @@ MachineEvaluation cross = machine_evaluation("CrossValidation", machine=svm, fea
 #![create_cross_validation]
 
 #![create_observer]
-ParameterObserverCV mkl_obs(True)
+ParameterObserver mkl_obs = parameter_observer("ParameterObserverCV")
```

Comment:
This works perfectly. However, some problems arises when trying to get an observation out of the observer:
```c++
CrossValidationStorage obs = mkl_obs.get_observation(0)
CrossValidationFoldStorage fold = obs.get_fold(0)
```
The call to ```mkl_obs.get_observation(0)``` extract one `Any` value. Is there any way to cast it somehow to its proper type by using the meta language? :/ @karlnapf any ideas?

Reply:
The way to do that is via ‘get’ which is available for all registered parameters

Reply:
so The objects that are exposed to the api need to have parameters registered with the appropriate type. Seems like we are pushing the problem around, need to think a bit about this

Reply:
See above discussion.